package.json

```diff
@@ -10,7 +10,13 @@
 		"wakamai-fondue": "./cli.js"
 	},
 	"files": [
-		"cli.js"
+		"cli.js",
+		"README.md"
+	],
+	"keywords": [
```

Comment:
@RoelN any other keywords we should add?

Reply:
I think these would be nice to add:

- variable fonts
- truetype
- otf
- ttf
- woff
- woff2
- css

Reply:
Added!